lib/whitehall_importer/integrity_checker.rb

```diff
@@ -87,10 +88,72 @@ def organisation_problems
       problems
     end
 
+    def unpublishing_problems
+      problems = []
+      return problems unless edition.withdrawn? || edition.removed?
```

Comment:
This feels like a bigger check to ensure all states match what's in Publishing API.  I'd like to keep this in place to avoid generating a problem for all editions imported that aren't unpublished, and do a separate piece of work to check for states in general.

Reply:
Ok - will there be a follow up story for this or separate PR in this story?

Reply:
I'll write up a story for it.